attractions/slider/handle.svelte

```diff
@@ -0,0 +1,150 @@
+<script>
+  import { calcPercentOfRange } from './utils';
+  import { tweened } from 'svelte/motion';
+  import { sineOut } from 'svelte/easing';
+  import { handleStyle } from './actions';
+
+  /**
+   * @type {number}
+   */
+  export let value;
+  /**
+   * @type {number}
+   */
+  export let min;
+  /**
+   * @type {number}
+   */
+  export let max;
+  /**
+   * @type {number}
+   */
+  export let index = 0;
+  /**
+   * @type {number}
+   */
+  export let activeHandle;
+  /**
+   * @type {boolean}
+   */
+  export let disabled;
+  /**
+   * @type {number}
+   */
+  export let tabIndex = index;
+  /**
+   * @type {boolean}
+   */
+  export let rectangular = false;
+  /**
+   * @type {'horizontal' | 'vertical'}
+   */
+  export let orientation = 'horizontal';
+
+  /**
+   * @type {boolean}
+   */
+  let focus = false;
+  /**
+   * @type {boolean}
+   */
+  let hovered = false;
```

Comment:
You requested earlier I make tooltip a slot, which is fine but now I need to compute and pass to the tooltip some state so it can decide what to do with it.